pom.xml

```diff
@@ -642,7 +649,6 @@
              <showWarnings>true</showWarnings>
              <compilerArgs>
                <compilerArg>-Werror</compilerArg>
-               <compilerArg>-Xlint:deprecation</compilerArg>
```

Comment:
Any reason to remove this? I had another PR not so long ago where after deprecating a function supposedly not used caused the build to fail, it was useful. We can remove it, warning is also fine for me, just not sure why to remove it in this patch, or why it has been decided so.

Reply:
All the important jUnit methods got deprecated and if we don't remove this line the build fails. (In this phase I use the jUnit4 compatibility mode.)
After the test code will have been converted to use jUnit5 methods (so not running in compatibility mode anymore) this line must be put back.
(This is why I commented it out originally instead of removing.)

Reply:
@tamaashu Please create a Blocker issue for restoring this value.

Reply:
Done. See ZOOKEEPER-3862